src/utils/printResult.js

```diff
@@ -0,0 +1,9 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+
+export const printResult = carsInfo => {
+	for (const car of carsInfo) {
```

Comment:
```suggestion
	for (const [name, count] of carsInfo) {
```
`car[0]`, `car[1]` 이렇게 사용하기 보다는 조금 더 명확한 이름을 사용하면 코드의 가독성이 향상될 수 있을 것 같아요! 
다른 함수부분에도 해당되는 것 같은데 추가로 코멘트는 달지 않겠습니다!